how2meet/ui/pages/events.py

```diff
@@ -72,72 +77,100 @@ async def render_list():
-    frame("Event Home")
+    Args:
+        event_id (str): The unique ID of the event to be created.
 
-    # Get the event from the API
-    event = await api.get_event(event_id)
+    Returns:
+        None
+    """
 
-    with ui.column().classes("w-full justify-center pt-10"):
-        ui.label(f"{event.get('name')}").classes("text-5xl font-bold border-l-8 border-teal-500 pl-4")
-        with ui.column().classes("w-full justify-left border-l-8 border-amber-300 pl-4"):
-            ui.label(f"{event_dates_to_str(event)}").classes("text-xl")
-            ui.label(f"{event_times_to_str(event)}").classes("text-xl")
-        ui.label(f"{event.get('location')}").classes("text-xl border-l-8 border-orange-400 pl-4 pr-4")
-        ui.label(f"{event.get('description')}").classes("text-xl border-l-8 border-orange-600 pl-4 pr-4")
+    frame()
 
-    # Display the list of guests in refreshable cards
+    event_editor = EventEditor()
+    # Render the event editor in page and set routing for save and back buttons
+    await event_editor.render(
+        on_back=partial(ui.open,
+                        URL_EVENTS),
+        on_save=partial(ui.open,
+                        URL_EVENT_HOME.format(event_id=event_editor.event_id)))
+
+# must come after /create to avoid path order conflict
+@router.page(ROUTE_EVENT_HOME)
+async def event_home(event_id: uuid.UUID, client: Client):
+    """Detail page for a specific event"""
+        # Display the list of guests in refreshable cards
     @ui.refreshable
     async def render_guests():
         guests = await api.get_guests_from_event(event_id)
         with ui.column().classes("w-full"):
             # sort guests by name and status
             avatar_colors = [color for color in elements.PALETTES.values() if color != elements.PALETTES["dark"]]
-            guests_going = sorted([guest for guest in guests if guest.get("status") == "Yes"], key=lambda x: x.get("name"))
-            guests_not_going = sorted([guest for guest in guests if guest.get("status") == "No"], key=lambda x: x.get("name"))
+            guests_going = sorted([guest for guest in guests if guest.get("status") == StatusEnum.GOING.value], key=lambda x: x.get("name"))
+            guests_not_going = sorted([guest for guest in guests if guest.get("status") != StatusEnum.GOING.value], key=lambda x: x.get("name"))
             i = 0
             for guest in guests_going + guests_not_going:
                 with ui.row().classes("w-full justify-left items-center"):
                     ui.avatar(guest.get("name")[0], color=avatar_colors[i], text_color="white", size="5xl")
                     # green check if going, red x if not going
-                    ui.icon("check" if guest.get("status") == "Yes" else "close")\
+                    icons = {
+                        StatusEnum.GOING.value: "check",
+                        StatusEnum.NOT_GOING.value: "close",
+                        StatusEnum.MAYBE.value: "question_mark"
```

Comment:
This is a discussion for another time, but I've been thinking about not offering a `maybe` option. Or at least giving the organizer the option to disable it. Justification: `maybe`'s encourage, or at least allow for flakiness on plans. In my perfect world, people would either say yes or no and stick to it